plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityEquipment.java

```diff
@@ -12,21 +14,21 @@
 import com.denizenscript.denizen.utilities.BukkitImplDeprecations;
 import net.citizensnpcs.api.trait.trait.Equipment;
 import org.bukkit.Material;
+import org.bukkit.inventory.EquipmentSlot;
 import org.bukkit.inventory.ItemStack;
 
 public class EntityEquipment implements Property {
 
-    public static boolean describes(ObjectTag entity) {
-        return entity instanceof EntityTag
-                && ((EntityTag) entity).isLivingEntity();
+    public static boolean describes(EntityTag entity) {
+        return entity.isLivingEntity();
     }
```

Comment:
Don't think this are correct? this property is still in the old property format, so it should have the `ObjectTag` params and all afaik, and updating it would be a different PR.